### PR TITLE
feat: support filling uuid when loading data and uuid is missing/empty in some rows.

### PR DESCRIPTION
**Pull request by youngsofun**
I hereby agree to the terms of the CLA available at: https://docs.databend.com/dev/policies/cla/

## Summary

 fix dependency by the way:  `databend-common-sql` should not dep on `storages-xxx`， add 2 func in table context, maybe we can have better way to solve it in the further.  

## Tests

- [ ] Unit Test
- [x] Logic Test
- [ ] Benchmark Test
- [ ] No Test - _Explain why_

## Type of change

- [x] Bug Fix (non-breaking change which fixes an issue)
- [ ] New Feature (non-breaking change which adds functionality)
- [ ] Breaking Change (fix or feature that could cause existing functionality not to work as expected)
- [ ] Documentation Update
- [ ] Refactoring
- [ ] Performance Improvement
- [ ] Other (please describe):

- - -
This change is [Reviewable](https://reviewable.io/reviews/datafuselabs/databend/15871)

### CODE REVIEW

**Comment by youngsofun** [comment]
cc @wubx

File: src/query/expression/src/expression.rs [hunk 1]
```diff
@@ -611,3 +611,11 @@ impl<Index: ColumnIndex> RemoteExpr<Index> {
         }
     }
 }
+
+#[derive(serde::Serialize, serde::Deserialize, Clone, PartialEq, Eq)]
+pub enum FieldDefaultExpr {
```
Comment (b41sh) [inline src/query/expression/src/expression.rs:616]: how about use `RemoteExpr` instead, which is more general.

**Review by b41sh — APPROVED** [review]

File: src/query/expression/src/expression.rs (hunk 1 above)
Comment (youngsofun) [reply src/query/expression/src/expression.rs:616]: good idea, but we need this fix as soon as possible, while I am a little busy now,  maybe we can refactor it later.

Comment (b41sh) [reply src/query/expression/src/expression.rs:616]: Since the PR needs are very urgent, we can first merge it and optimize codes later.

**Pull request merged**